uwblueprint/childrens-aid-society
# create dummy referrals screen

## Notion ticket link

[Implement dummy referral details screen](https://www.notion.so/uwblueprintexecs/Task-Board-db95cd7b93f245f78ee85e3a8a6a316d)



## Implementation description
* created a new referral screen using Chakra UI's form control
* added props for referralDetails & setReferralDetails that will be modified when we input info into our form
* those props are passed in from IntakePage so that we can have access to the data there



## Steps to test
1. checkout the branch
2. start up docker containers
3. go to localhost:3000/intake, and check if the Referral Forms screen exists on intake 1
4. if you want to double check the correct info is at the IntakePage.tsx file, you can add in console.log('details', referralDetails)

NEW STYLING HERE :DDD 😁
![Screen Shot 2022-07-13 at 9 10 51 PM](https://user-images.githubusercontent.com/77185514/178864682-2a0f406e-9021-45b2-aefb-774a8a2859b2.png)




## What should reviewers focus on?
* whether or not the styles file should be there/ if the styling is correctly structured
* whether or not the typescript format for passing in props is correct (it didn't seem to work when i passed in referralDetails & setReferralDetails manually without React.FC

## Checklist
